modules/loki-alerts.adoc

```diff
@@ -13,6 +13,11 @@ The following alerts are generated by the {loki-op}. You can view these alerts i
 |===
 |Alert |Message |Description |Severity
 
+|`LokiIngesterFlushFailureRateCritical`
+a|`Loki ingester {{ $labels.pod }} in the namespace {{ $labels.namespace }} has a critical flush failure rate of {{ $value \| humanizePercentage }} over the last 5 minutes. This requires immediate attention as data is not being flushed to the storage. Validate if the storage configuration is still valid and if the storage is still reachable. Current failure rate: {{ $value \| humanizePercentage }} Threshold: 20%`
+|One or more Loki ingesters are failing to flush at least 20% of their chunks to the storage over a five-minute period. This indicates issues with storage connectivity, authentication, or capacity that require immediate intervention.
```

Comment:
Suggestion (keeps in line with grafana repo):
One or more Loki ingesters are failing to flush at least 20% of their chunks to **backend** storage over a **5**-minute period. This indicates issues with storage connectivity, authentication, or storage capacity that require immediate intervention.